include/oneapi/dpl/numeric

```diff
@@ -43,6 +43,27 @@
 #    include "oneapi/dpl/pstl/glue_numeric_impl.h"
 #endif
 
+namespace oneapi
+{
+namespace dpl
+{
+using ::std::accumulate;
```

Comment:
@akukanov @capatober
We are adding all "Sequential" version of numeric APIs in oneDPL which are tested. Please help review.
Currently, there is one problem, some old version libstdc++ doesn't include some of those APIs which will lead to compfail when we try to inject them to oneapi::dpl namespace. 
Checking libstdc++/MSVC version and only do the injection when those APIs does exist in system seems to work, do you have any better idea?
Thank very much.

Reply:
Looks good to me. We usually do libstdc++ macros checking. See an example: https://github.com/oneapi-src/oneDPL/blob/master/include/oneapi/dpl/pstl/onedpl_config.h#L265